### PR TITLE
Should provide the scale of a single axis

### PR DESCRIPTION
**Pull request by minoism**

### CODE REVIEW

**Comment by ReinBentdal** [comment]
Thanks for the contribution! I am currently busy with exams. I will come back and take a look when i got time

File: lib/src/extensions/widget_extension.dart [hunk 1]
```diff
@@ -727,8 +727,10 @@ extension StyledWidget on Widget {
               transformHitTests: transformHitTests,
             );
 
-  Widget scale(
-    double scale, {
+  Widget scale({
+    double all,
```
Comment (ReinBentdal) [inline lib/src/extensions/widget_extension.dart:731]: Consider changing `all` to `proportional`? Worth the longer word to make it more descriptive?

**Review by ReinBentdal — CHANGES_REQUESTED** [review]

File: lib/src/extensions/widget_extension.dart (hunk 1 above)
Comment (minoism) [reply lib/src/extensions/widget_extension.dart:731]: I think the fine, but short word may help keep the code concise, just like `EdgeInsets.all`?

Comment (ReinBentdal) [reply lib/src/extensions/widget_extension.dart:731]: Okey, i will accept your changes. Thanks for contributing to the project!

**Pull request merged**